Setup/UpgradeSchema.php

```diff
@@ -75,7 +75,7 @@ public function upgrade(
             }
         }
 
-        $tableName = $setup->getTable(OrderLink::TABLE_NAME);
+        $tableName = $setup->getTable('sales_order');
```

Comment:
This is not directly related to the purpose of your PR, I believe it would be better to roll back this specific line change, as it slows down your PR processing.